.github/workflows/build.yml

```diff
@@ -0,0 +1,79 @@
+name: Jasper reporting service JAR Build
+on: [push]
+
+jobs:
+    build:
+      runs-on: ubuntu-latest
+      steps:
+      - uses: actions/checkout@v2
+
+      - name: Create java enviroment
+        uses: actions/setup-java@v1
+        with:
+            java-version: '1.8'
+
+      - name: Install fonts.jar
+        run: |
+          mvn install:install-file -Dfile=demo/fonts/fonts-extension.jar -DgroupId=local.jasperFontOverrides -DartifactId=local.jasperFontOverrides -Dversion=1.0 -Dpackaging=jar -DlocalRepositoryPath=$HOME/.m2/repository
+      - name: Build jar with maven
+        run:
+          mvn package -Dmaven.wagon.http.ssl.ignore.validity.dates=true -Dmaven.wagon.http.ssl.allowall=true -Dmaven.wagon.http.ssl.insecure=true
+
+      - name: Version number
+        id: version_number
+        run: |
+          if [ ${{ startsWith(github.ref, 'refs/tags/') }} = true ]; then
+            VERSION=$(basename ${{ github.ref }})
+          else
+            VERSION=ci-latest-$(basename ${{ github.ref }})
+            PREV_RELEASE=$(curl -s https://api.github.com/repos/$GITHUB_REPOSITORY/releases/tags/$VERSION | jq -r .url)
```

Comment:
Make separate step as in https://github.com/kadas-albireo/kadas-albireo2/blob/master/.github/workflows/build.yml#L46